samples/cellular/modem_shell/src/ping/icmp_ping.c

```diff
@@ -52,6 +54,15 @@ enum ping_rai {
 	PING_RAI_LAST_PACKET
 };
 
+int64_t reg_time; /* When LTE reg status was updated last time:
```

Comment:
This should probably be static? And naming maybe a bit more detailed to contain some part or all of `network_reg_status_updated_uptime`.

Reply:
Yes, and renamed